### ISSUE TITLE
Pick a more accessible link colour

### ISSUE THREAD

**Issue by zarino** [official-inquiries/official-inquiries.com#14]
Sorry to be _one of those people_ but when I saw this Table of Contents at the top of the Chillcot exec summary, I had a really hard time actually making out any of the words:

![screen shot 2016-07-11 at 08 59 29](https://cloud.githubusercontent.com/assets/739624/16723590/ca69a836-4745-11e6-8d88-2e10016cb0a1.png)

The WCAG AA standard is usually a good indicator of "is this thing legible or not" – your link colour (`#7ab800`) on a white background is a fail.

Darkening it to something like `#558000` would make it easier to read, without losing the "green-ness":

![screen shot 2016-07-11 at 09 01 56](https://cloud.githubusercontent.com/assets/739624/16723642/2c18a49c-4746-11e6-997a-6197d76ee751.png)

**Comment by zarino** [comment]
(PS. Sorry if this is the wrong place to report this – I couldn't find a link to the source code on your site, so I had to guess this was the right repo)

**Comment by GregoryRhysEvans** [comment]
@zarino Thanks very much for the heads-up. This is indeed the right place to raise the issue and we'll take a look at this; and, of course, we welcome any further input you may have.
